index.html

```diff
@@ -13,10 +13,10 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css"
     />
     <link rel="stylesheet" href="main.css" />
-    <title>Petmate | Login</title>
+    <title>Petmate | Landing Page</title>
```

Comment:
I think there is no need to write landing page in title because its obvious its your landing page

Reply:
ok will make the changes. Thank you.